src/main/java/io/vertx/codegen/type/TypeVariableInfo.java

```diff
@@ -48,7 +48,7 @@ public boolean isNullable() {
 
   @Override
   public TypeInfo getErased() {
-    return new ClassTypeInfo(ClassKind.OBJECT, Object.class.getName(), null, false, Collections.emptyList());
+    return new ClassTypeInfo(ClassKind.OBJECT, Object.class.getName(), null, false, Collections.emptyList(), null);
```

Comment:
Why this `TypeVariableInfo` doesn't contain `TypeInfo` but `TypeParamInfo`? To me it sounds like a potential inconsistency problem, in particular when reading `TypeParam`s of `List` and `Map`. Or do I miss something?

Reply:
it is not clear what you mean, can you elaborate and give example ?

Reply:
I've checked again, i misunderstood the meaning of this class